### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (10 - Text Alternatives

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#489]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

[Note, see the original document for screenshots]

7.1 text alternatives
 
This starts out with an imperative or directive rather than an outcome. 
The imperative is "Provide text alternatives for all non-text content”.  
This would be comparable to the WCAG 2 Guideline 1.1 text alternatives: provide text alternatives for any non-text content so that it can be changed into other forms people need such as large print braille, speech, symbols or simple language.
This is then followed in WCAG 3.0 with the outcome "text alternative available”
This is a good statement of an outcome.
However this is then followed by something which says 
" Provides text alternatives for non-text content for user agents and assistive technologies. This allows users who are unable to perceive and / or understand the non-text content to determine its meaning”
  
This text is unlabeled  (as is the Objective).  If this text was labeled as benefit or Rationale than this overall 7.1 provision would be pretty good.  
1.	It has a general sentence above (Provide text alternatives.. ) that is not-normative but helps the user understand the objective
a.	 This is similar to the WCAG 2 ‘Guidelines” which were also non-normative.  I would suggest though that if reformatting – we might look to eliminating imperatives and focusing only on outcome-oriented statements (see comments above on future resilience in the Laudits section)  
2.	It would have a very simple outcome-oriented statement which is testable "text alternative available" – and would be normative.
a.	(It could be a bit less ambiguous since it seems to require that there be a text alternative for text.  It would probably be better if it read "text alternative available for any nontext content" – or easier to read by inserting the word is “A text alternative is available for any non-text content”.
b.	This should be clearly labeled or in some way clear that it is the “Desired Outcome” or “Requirement”.  It currently is only labeled as such in the markup – which requires you look at the source to discover this.  Using visual formatting to indicate that this is the key phrase – violates 1.3.1 of WCAG. So, some text labelling should be provided to assist both those with visual disabilities and those who have trouble processing complex documents sort out the key/core text of the provision. 
3.	It would then have a Rationale that explains the purpose and benefit of the provision making it easier to understand and providing motivation to conform. 
a.	This however should be labelled as such – as noted above. 
b.	This would be non-normative. 
Note that the comments about users being able to understand etc. are all in the rationale and so they do not trip the concerns I cited above about discussing users within actuals standards normative comments.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by MakotoUeki** [comment]
DRAFT Response (25th May) by Alt text sub-group:
Thank you for your comments and suggestions on the first public working draft and the “7.1 Text alternatives”.

We will need to be consistent with the rest of the guidelines, but we agree with you on modifying the guideline wording to “A text alternative is available for any non-text content”. It is less ambiguous.

We agree that adding section headers would be helpful. We will discuss this issue and your concerns regarding directive versus outcome language with the larger group.

Let us work on the future versions of our working draft to address your concerns. It would be very helpful if you could keep watching them and sharing your thoughts on our future modifications.
